big_tests/test.config

```diff
@@ -285,7 +285,7 @@
                                                             {keyfile, \"priv/ssl/fake_key.pem\"}]}]}
        ]}."},
       {mod_offline, "{mod_offline, []},"},
-      {cyrsasl_external, "{cyrsasl_external, standard}"}, % just to put a term before 'auth_ldap'
+      {cyrsasl_external, ", {cyrsasl_external, standard}"}, % just to put a term before 'auth_ldap'
```

Comment:
I'm pretty sure that if we know that the `auth_opts` list contains always at least the `password_format` there is no need for this hack.

Reply:
Sure, I just don't know the exact history of this line and was not eager to have it removed straight away. But if suggest having it removed, will have that changed.